### DISCUSSION TITLE
Matching Repos: Ender3V2S1 & Special_Configurations

### DISCUSSION THREAD

**Discussion by Homer-S** [mriscoc/Ender3V2S1#1168] [Q&A]
Trying to compile the recent release, awesome firmware BTW, but am having no luck matching the versions of these two repos.

Are these meant to go together?

![image](https://github.com/mriscoc/Ender3V2S1/assets/1320262/4426c0c0-675d-449b-8832-8f5e3d13e28f)
![image](https://github.com/mriscoc/Ender3V2S1/assets/1320262/98be4c2b-c38e-445a-bfaf-5ddf26f27a77)

Generically, how do we match the Special Configuration file version to an arbitrary release version?

Thanks!
Homer

**Comment by mriscoc** [comment]
It is usually is enough to download the Special Configuration around the date of the main repository.

**Reply by Homer-S** [reply]
I *think* I get that. For the recent release "DEMO", I can't get it to compile. they seem like they should go together being dated the same day, 20231202, but I'm getting errors. First the one that talks about a custom cable for the BTT board and if I define the "define" to bypass that error I get a cascade of missing stuff. I get the same cascade of errors from the previous release as well.

Homer
